coding-intro/coding-intro-core/core-concepts/think-like-a-coder.md

```diff
@@ -0,0 +1,49 @@
+---
+author: abdullah
+
+tags:
+  - discussion
+
+type: normal
+
+category: discussion
+
+---
+
+
+# Think like a coder
+
+---
+## Content
+
+Coding is all about saving time (and sometimes about making money 😉).
+
+Let's look at a scenario.
+
+Imagine you have to simulate grabbing 5 coins from 3 different jars. 
+
+You could make a function, `pick_coins`, with 3 commands and use it 5 times, saving yourself a lot of time.
+
+```python
```

Comment:
you set the language tag to `python` but you're using curly braces when defining functions

```suggestion
```js
```